requirements.txt

```diff
@@ -6,4 +6,7 @@ pillow>=11.2.1
 requests>=2.32
 pychromecast~=14.0.7
 httpx~=0.28.1
-sanitize-filename~=1.2.0
\ No newline at end of file
+sanitize-filename~=1.2.0
+async_timeout~=5.0.1
+adb_shell==0.4.4
+simple-justwatch-python-api>=0.16
```

Comment:
The https://github.com/Electronic-Mango/simple-justwatch-python-api library is licensed under GPLv3. Unfortunately we cannot include this in our device firmware.
Please either remove it or replace it with another library licensed under BSD, MIT, Apache-2.0 or MPL-2.0